### PR TITLE
Fix/bddl errors

### PR DESCRIPTION
**Pull request by ChengshuLi**

### CODE REVIEW

**Comment by ChengshuLi** [comment]
@cremebrule Could you take a look at the PR and let's get this merged?

File: bddl/activity_definitions/cleaning_around_pool_in_garden/problem0.bddl [hunk 1]
```diff
@@ -13,7 +13,7 @@
     (:init 
         (covered driveway.n.01_1 dust.n.01_1) 
         (ontop broom.n.01_1 floor.n.01_1)
-        (ontop pool.n.01_1 driveway.n.01_1) 
+        (inroom pool.n.01_1 garden)
```
Comment (sanjanasrivastava) [inline bddl/activity_definitions/cleaning_around_pool_in_garden/problem0.bddl:16]: The goal conditions no longer make sense given the task description

File: bddl/activity_definitions/cleaning_cups_in_living_room/problem0.bddl [hunk 2]
```diff
@@ -20,9 +20,9 @@
         (covered wineglass.n.01_1 dust.n.01_1) 
         (covered wineglass.n.01_2 dust.n.01_1) 
         (covered wineglass.n.01_3 dust.n.01_1) 
-        (covered wineglass.n.01_1 red_wine.n.01_1)
-        (covered wineglass.n.01_2 red_wine.n.01_1)
-        (covered wineglass.n.01_3 white_wine.n.01_1)
+        (filled wineglass.n.01_1 red_wine.n.01_1)
+        (filled wineglass.n.01_2 red_wine.n.01_1)
+        (filled wineglass.n.01_3 white_wine.n.01_1)
```
Comment (sanjanasrivastava) [inline bddl/activity_definitions/cleaning_cups_in_living_room/problem0.bddl:25]: This is fine, but I do think it was better before for a task about cleaning - is there a problem with covered?

File: bddl/activity_definitions/cleaning_around_pool_in_garden/problem0.bddl (hunk 1 above)
Comment (cremebrule) [reply bddl/activity_definitions/cleaning_around_pool_in_garden/problem0.bddl:16]: hmmmm I see your point. Is there any clear way to fix this? l suppose there's no way to explicitly capture the "near pool" semantics in bddl-language, right?

File: bddl/activity_definitions/cleaning_cups_in_living_room/problem0.bddl (hunk 2 above)
Comment (cremebrule) [reply bddl/activity_definitions/cleaning_cups_in_living_room/problem0.bddl:25]: Sampling was failing, but now with some changes Eric and I have made, covered might be fine. I can revert this

Comment (cremebrule) [reply bddl/activity_definitions/cleaning_cups_in_living_room/problem0.bddl]: reverted

File: bddl/activity_definitions/cleaning_around_pool_in_garden/problem0.bddl (hunk 1 above)
Comment (cremebrule) [reply bddl/activity_definitions/cleaning_around_pool_in_garden/problem0.bddl:16]: conclusion: pool being in same room is sufficient

**Pull request merged**